modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2550,6 +2551,12 @@
 		end
 	end
 
+	function m.enableJumboBuild(cfg)
+		if _ACTION > "vs2017" and cfg.enableUnityBuild then
```

Comment:
```suggestion
		if _ACTION >`= "vs2019" and cfg.enableUnityBuild then
```
To be coherent with surrounding conditions.

Reply:
Ah yes my bad, unity builds exist on 2017, updated the docs.

Reply:
So 
```suggestion
		if _ACTION >= "vs2017" and cfg.enableUnityBuild then
```